release-notes/azure/v11.1.0.md

```diff
@@ -64,16 +64,14 @@ In order to ensure continuous service during upgrade, the workload must be confi
 
 - No change from the previous version
 
-### net-exporter [v1.6.0](https://github.com/giantswarm/net-exporter/blob/master/CHANGELOG.md#160-2020-01-29)
+### net-exporter [v1.5.1](https://github.com/giantswarm/net-exporter/blob/master/CHANGELOG.md#151-2020-01-08)
 
-- Allow disabling DNS TCP check.
+- No change from the previous version
 
-### nginx-ingress-controller v0.28.0 ([Giant Swarm app v1.4.0](https://github.com/giantswarm/nginx-ingress-controller-app/blob/master/CHANGELOG.md#v140-2020-02-10))
+### nginx-ingress-controller v0.26.1 ([Giant Swarm app v1.1.1](https://github.com/giantswarm/nginx-ingress-controller-app/blob/master/CHANGELOG.md#v111-2020-01-04))
```

Comment:
@axbarsan did you maybe already prepare moving these into 11.2.0 or should I do it?

Reply:
I converted [this PR](https://github.com/giantswarm/releases/pull/84) to be the 11.2.0 release, since we already had so many changes on it

Reply:
That was clear. Problem is that the release notes, details, being removed from 11.1.0 with this PR, are not present in 11.2.0, should probably be moved to 11.2.0.

Reply:
That's what the other PR is for, I added them there